elements/simple-login/lib/simple-login-camera.js

```diff
@@ -171,36 +171,46 @@ class SimpleLoginCamera extends HTMLElement {
   }
   _msrLoaded(e) {
     this._applyMSR();
+    this._shadow.querySelector("video").addEventListener("click",()=>{
+      this.dispatchEvent(
+        new CustomEvent("camera-icon", {
+          detail: this,
+          bubbles: true,
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Event/composed
See `composed: true` to ensure that this bubbles OUT of shadowRoots.

Great article that I just saw posted that I am including in class next semester -- https://pm.dartus.fr/blog/a-complete-guide-on-shadow-dom-and-event-propagation/